dist/init/linux-systemd/README.md

```diff
@@ -6,26 +6,153 @@ if you have any questions. Feel free to prepend to your question
 the username of whoever touched the file most recently, for example
 `@wmark re systemd: …`.
 
-The provided file should work with systemd version 219 or later. It might work with earlier versions.
+The provided file should work with systemd version 229 or later. It might work with earlier versions.
 The easiest way to check your systemd version is to look at the version of the installed package
 (e.g. 'sudo yum info systemd' on RedHat/Fedora systems).
 
-## Instructions
-
 We will assume the following:
 
-* that you want to run caddy as user `www-data` and group `www-data`, with UID and GID 33
+* that you want to run caddy as user `www-data` and group `www-data`
 * you are working from a non-root user account that can use 'sudo' to execute commands as root
 
 Adjust as necessary or according to your preferences.
 
-First, put the caddy binary in the system wide binary directory and give it
-appropriate ownership and permissions:
+## Install
+
+There's a full caddy install that works with systemd over at https://git.daplie.com/Daplie/caddy-installer
+if you'd like to try that out.
 
 ```bash
-sudo cp /path/to/caddy /usr/local/bin
-sudo chown root:root /usr/local/bin/caddy
-sudo chmod 755 /usr/local/bin/caddy
+# install caddy with a few features
+curl -L https://git.daplie.com/Daplie/caddy-installer/raw/master/install-caddy | bash -s -- minify,realip
+```
+
+Otherwise just fetch `caddy.service` and `caddy.conf` and put them in the right places:
+
+```bash
+curl -sL https://raw.githubusercontent.com/mholt/caddy/master/dist/init/linux-systemd/caddy.service -o caddy.service
+curl -sL https://raw.githubusercontent.com/mholt/caddy/master/dist/init/linux-systemd/caddy.conf -o caddy.conf
+
+sudo chown root:www-data caddy.service
+sudo chown root:www-data caddy.conf
+
+sudo chmod 0755 caddy.service
+sudo chmod 0755 caddy.conf
+
+sudo mv caddy.service /etc/systemd/system/
+sudo mv caddy.conf /etc/tmpfiles.d/
```

Comment:
Caddy is the webserver for beginners... I mean it advertises on the homepage using PHP... which is extremely ironic considering the the focus on security...